frontend/benefit/applicant/src/components/applications/forms/application/step4/attachmentsListView/AttachmentsListView.tsx

```diff
@@ -0,0 +1,42 @@
+import { ATTACHMENT_TYPES } from 'benefit/applicant/constants';
+import { Attachment } from 'benefit/applicant/types/application';
+import { IconPaperclip } from 'hds-react';
+import * as React from 'react';
+
+import {
+  $ViewField,
+  $ViewFieldBold,
+  $ViewFieldsContainer,
+  $ViewFieldsGroup,
+} from '../../Application.sc';
+
+export interface AttachmentsListViewProps {
+  attachments: Attachment[];
+  type: ATTACHMENT_TYPES;
+  title: string;
+}
+
+const AttachmentsListView: React.FC<AttachmentsListViewProps> = ({
+  attachments,
+  type,
+  title,
+}) => (
+    <$ViewFieldsContainer>
+      <$ViewFieldsGroup>
+        <$ViewFieldBold>{title}</$ViewFieldBold>
+        {attachments
```

Comment:
Summery doesnt change much so I was wondering that this could be wrapped with `React.memo(...)` but it could be premature optimizing also.

Reply:
true that